src/Evolution/Initialization/InitialData.hpp

```diff
@@ -31,4 +32,38 @@ decltype(auto) initial_data(const SolutionOrData& solution_or_data,
                                     tags);
 }
 /// \endcond
+
+
+// Wrap a new initial_data function within ScalarAdvection namespace
+namespace ScalarAdvection {
```

Comment:
remove the namespace. I think the new function shouldn't collide with the existing ones, but not entirely sure.